File: playground/php/src/personalization.php
<?php

$env = require_once('../loadEnv.php');

use Algolia\AlgoliaSearch\Api\PersonalizationClient;

$client = PersonalizationClient::create($env['ALGOLIA_APPLICATION_ID'], $env['ALGOLIA_RECOMMENDATION_KEY']);

var_dump(
    $client->deleteUserProfile('userToken')
);

File: playground/go/personalization.go
package main

import (
	"fmt"

	"github.com/algolia/algoliasearch-client-go/v4/algolia/personalization"
)

func testPersonalization(appID, apiKey string) int {
	personalizationClient := personalization.NewClient(appID, apiKey, personalization.US)

	deleteUserProfileResponse, err := personalizationClient.DeleteUserProfile(
		personalizationClient.NewApiDeleteUserProfileRequest("userToken"),
	)
	if err != nil {
		fmt.Printf("request error with DeleteUserProfile: %v\n", err)
		return 1
	}

	printResponse(deleteUserProfileResponse)

	return 0
}
